src/integrationTest/java/uk/gov/hmcts/reform/translate/controllers/DictionaryControllerIT.java

```diff
@@ -73,10 +78,39 @@ void shouldThrowExceptionWhenDictionaryReturnsDuplicateEnglishPhrases() {
         @Test
         void shouldReturn403WhenUserDoesNotHaveManageTranslationsRole() throws Exception {
             stubUserInfo("unknown-role");
-            mockMvc.perform(get(URL)
+            mockMvc.perform(get(DICTIONARY_URL)
                                 .contentType(MediaType.APPLICATION_JSON_VALUE))
                 .andExpect(status().is(403))
                 .andReturn();
         }
     }
+
+    @Nested
+    class GetTranslations {
+        private static final String TRANSLATIONS_URL = "/translation/cy";
+
+        @ParameterizedTest
+        @EmptySource
+        @ValueSource(strings = {"{}", "{\"phrases\":[]}", "{\"phrases\":[\"\"]}", "{\"illegal\":[\"English Phrase\"]}"})
+        void shouldReturn400BadRequestWhenBadTranslationRequestIsSubmitted(final String input) throws Exception {
+            mockMvc.perform(post(TRANSLATIONS_URL)
+                                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                                .accept(MediaType.APPLICATION_JSON_VALUE)
+                                .content(input))
+                .andExpect(status().isBadRequest());
+        }
+
+        @Test
+        @Sql(scripts = {DELETE_TRANSLATION_TABLES_SCRIPT, GET_TRANSLATION_TABLES_SCRIPT})
+        void shouldReturn200WhenLegalRequestIsSubmitted() throws Exception {
+            final Map<String, String> expectedTranslations = Map.of("English Phrase 2", "Translated Phrase 2");
```

Comment:
(Although already tested in unit tests and FTAs) Is it worth having multiple phrases here in this one test, including
* phrase with translation,
* phrase without,
* a new phrase (i.e. not in DB script)

Reply:
All these scenarios are covered in `DictionaryServiceTest`.  I don't see any value in repeating them at the controller level and again in the Functional tests

Reply:
Agreed covered e2e in FTAs under #42.  Therefor setting to resolved.